package.json

```diff
@@ -19,6 +19,7 @@
     "date-fns": "^2.14.0",
     "diff": "^5.0.0",
     "font-awesome": "^4.7.0",
+    "gcc": "^1.0.2",
```

Comment:
Could you please confirm if this is intentional?

Reply:
No thjis was an error on my end